### ISSUE TITLE
Unable to cast type of "dotless.Core.Parser.Tree.Ruleset" to type "dotless.Core.Parser.Tree.Ruleset"

### ISSUE THREAD

**Issue by AjithR04** [dotless/dotless#488]
I have used guards in my less css with OR predicate. When I tried to compile the less using dotLess 1.5.1.0 compiler it throws the compilation error. 

Please refer the below code snippet.

Car.less
@var: "";
& when (@var = TATA ), (@var = "" ){
     /*Tata*/
     .t-car {
          color: white;
     }
     .t-bg {
          background: blue;
     }
}

![image](https://cloud.githubusercontent.com/assets/13574028/9732632/f67d96e8-5642-11e5-8c31-93d380ae7356.png)

**Comment by rytmis** [comment]
Yeah, I don't think we support that kind of predicate currently. I'll throw it on the todo-list for 1.6. :)
